### ISSUE TITLE
Update Kotlin to `1.8.0`

### ISSUE THREAD

**Issue by YektaDev** [joreilly/PeopleInSpace#185]
**Kotlin 1.8.0** [is out](https://github.com/JetBrains/kotlin/releases/tag/v1.8.0/)!
Although it might take some time for the compose compiler to be compatible with this version, I thought it might be good to have an issue regarding this topic.

**Comment by joreilly** [comment]
Also waiting on SqlDelight 1.5.5 which has some 1.8 related changes as well.

**Comment by joreilly** [comment]
Waiting specifically on https://github.com/cashapp/sqldelight/issues/3738 (which is in 1.5.5)

**Comment by YektaDev** [comment]
Well, and now there's the [1.5.5](https://github.com/cashapp/sqldelight/releases/tag/1.5.5) release🎉

**Comment by joreilly** [comment]
There is indeed 😄 ....have branch using it now at https://github.com/joreilly/PeopleInSpace/compare/kotlin_1_8?expand=1. (though looking in to another issue right now)
